src/python/pants/backend/python/goals/pytest_runner_integration_test.py

```diff
@@ -418,3 +444,22 @@ def test_args():
     extra_env_vars = '["SOME_VAR=some_value", "OTHER_VAR"]'
     result = run_pytest(rule_runner, tgt, extra_env_vars=extra_env_vars, pants_environment=mock_env)
     assert result.exit_code == 0
+
+
+def test_binary_dependency(rule_runner: RuleRunner) -> None:
+    create_python_binary_target(rule_runner, BINARY_SOURCE)
+
+    test_source = FileContent(
+        path=f"{PACKAGE}/test_binary_call.py",
+        content=dedent(
+            """\
+            import subprocess
+
+            def test_embedded_binary():
+                assert  b"Hello, test!" in subprocess.check_output(args=['./bin.pex'])
+            """
+        ).encode(),
+    )
+    tgt = create_test_target(rule_runner, [test_source], dependencies=[":bin"])
+    result = run_pytest(rule_runner, tgt, passthrough_args="-s")
```

Comment:
No need for the passthrough args afaict.

Reply:
It is necessary, by default pytest doesn't display the output of passing tests.